src/assets/stylesheets/main/extensions/pymdownx/_highlight.scss

```diff
@@ -322,22 +322,43 @@
   }
 
   // Code block result container
-  :is(.highlight, .highlighttable) + .result {
-    margin-top: calc(-1em + #{px2em(-2px)});
-    padding: 0 px2em(16px);
-    overflow: visible;
-    border: px2rem(1px) solid var(--md-code-bg-color);
-    border-top-width: px2rem(2px);
-    border-bottom-right-radius: px2rem(2px);
-    border-bottom-left-radius: px2rem(2px);
-
-    // Clearfix, because we can't use overflow: auto
-    &::after {
-      display: block;
-      clear: both;
-      content: "";
+  // sphinx-immaterial: altered to support
+  // 1. docutils classes (.literal-block-wrapper, div[class^="highlight-"])
+  // 2. rst-result directive's output-prefix (.results-prefix)
+  :is(.highlight, .highlighttable, .literal-block-wrapper),
```

Comment:
I think you can put all of the conditions inside `:is`, e.g. `:is(.highlight, .highlighttable, div[class^="highlight-"], .results-prefix) + .result`

That way the indentation doesn't change so merging will be easier

Reply:
I think this request violates the stylelint line length though

Reply:
Nope. It seems to have passed stylelint checks 👍🏼